File: benchmarks/string/concat_100k.go
package main

import (
	"fmt"
	"strconv"
	"strings"
	"time"
)

func main() {
	parts := make([]string, 100000)
	for i := range parts {
		parts[i] = strconv.Itoa(i)
	}
	start := time.Now()
	_ = strings.Join(parts, "")
	fmt.Println(time.Since(start).Microseconds())
}

File: benchmarks/string/concat_100k.rs
use std::time::Instant;
fn main() {
    let parts: Vec<String> = (0..100000).map(|i| i.to_string()).collect();
    let start = Instant::now();
    let _result: String = parts.join("");
    println!("{}", start.elapsed().as_micros());
}
